PR_Standard.md

```diff
@@ -0,0 +1,35 @@
+# Description
```

Comment:
The Checklist template looks good. It needs to be updated in some areas to reflect the project and also need the MD syntax updated.